ipam/space/space_test.go

```diff
@@ -19,6 +19,15 @@ func ip(s string) address.Address {
 	return addr
 }
 
+// Helper function to avoid editing dozens of lines of test
```

Comment:
I f I thought that was the case I wouldn't have suggested keeping the function.

It's useful because the alternative is having dozens of lines invoking a function with the same two args.